### DISCUSSION TITLE
Problems when in SSR mode

### DISCUSSION THREAD

**Discussion by jonasbros** [quasarframework/quasar#8913] [CLI - SSR mode] [answered]
Hey guys, attached picture is the error I'm getting when running in SSR mode. Does it have to do with the Vuex store? Thanks a lot guys, I'm still pretty new to Quasar.

`quasar dev -m ssr`

![Capture](https://user-images.githubusercontent.com/24856951/114987952-59130a80-9ec8-11eb-9711-6cfbf6750130.PNG)

**Comment by rstoenescu** [comment, marked as answer]
The presence of the "src/store" folder indicates that you are using Vuex. If it was created by you for other purposes (and not created the project with Vuex option), then rename it to something else. Alternatively, tell Quasar CLI that the Vuex folder should be named as something else: quasar.conf.js > sourceFiles > store: 'set-some-bogus-content'
